src/coverup/coverup.py

```diff
@@ -403,6 +413,8 @@ async def do_chat(seg: CodeSegment, completion: dict) -> str:
 
     global token_rate_limit
 
+    # FIXME
+    token_rate_limit = None
```

Comment:
This would always disable the rate limit... is the issue that you don't want a rate limit used with bedrock?
If the model is unknown in `token_rate_limit_for_model()` (`src/coverup/llm.py`), no rate limit is used.
Anyway, you probably want to delete this (as it would break OpenAI usage).

Reply:
Removed.